src/main.rs

```diff
@@ -40,12 +41,19 @@ async fn main() {
 
     if options.subcommand_matches("list").is_some() {
         list::show_list(&scenarios);
+        std::process::exit(exitcode::OK)
     }
 
     if let Some(run_options) = options.subcommand_matches("run") {
         let scenario_label = run_options.value_of("SET").unwrap();
         let endpoint_url = run_options.value_of("url").unwrap();
         let api_key = run_options.value_of("API_KEY").unwrap();
-        run::run(endpoint_url, api_key, scenario_label.to_string(), scenarios).await;
+        match run::run(endpoint_url, api_key, scenario_label.to_string(), scenarios).await {
+            Ok(()) => std::process::exit(exitcode::OK),
+            Err(()) => std::process::exit(1)
```

Comment:
I'm pretty surprised `exitcode` doesn't have a `1` exit code inside, I know it just wraps value from `sysexits.h` but if feels a little clumsy :smile:

Reply:
Yeah man, i don't know why it doesn't include it. I was just to about to remove the complete package usage only for that hehe